src/components/Blogs.js

```diff
@@ -0,0 +1,50 @@
+import { useEffect, useState } from "react";
+import axiosInstance from "../api/axios";
+
+const Blogs = () => {
+  
+  const [posts, setPosts] = useState([]);
+  const [visible, setVisible] = useState(3);
+
+  const loadMore = () => {
+    setVisible((moreItem) => moreItem + 3);
+  };
+
+  useEffect(() => {
+    const getBlog = async () => {
+      try {
+        const results = await axiosInstance.get();
```

Comment:
Are you getting all that data at once?

Is the backend sending all the posts at once?

Reply:
yes. this particular function gets blogs from all authenticated user which is paginated.